client/src/app.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react';
 import axios from 'axios';
+import { Reset } from 'styled-reset';
```

Comment:
what does this do?

Reply:
That allows me to apply the style reset to all the CSS on the page. That will prevent browsers from applying any default styling.